### PR TITLE
Updated teamserver access to reflect new git infra

### PR DESCRIPTION
**Pull request by grootjans**

### CODE REVIEW

File: content/en/docs/refguide/version-control/version-control-troubleshooting/troubleshoot-team-server-issues.md [hunk 1]
```diff
@@ -9,30 +9,35 @@ tags: ["version control", "troubleshoot", "team server"]
 
 ## 1 Introduction
 
-Mendix Studio Pro needs to connect to the Team Server, where all your apps are stored. This document describes which permissions and settings are required to connect to the Team Server.
+Mendix Studio Pro needs to connect to the Team Server, where all your apps are stored. Team Server is a version control server. Currently, SVN is the default version control system for the Team Server, but you can choose to use Git. In the future, Git will become the default system. This document describes permissions and settings  required to connect to the Team Server.
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/version-control/version-control-troubleshooting/troubleshoot-team-server-issues.md:12]: > In the future, Git will become the default system.
If anyone asks, can we give approximate dates? Shall we leave this info out if we can't?

File: content/en/docs/refguide/version-control/version-control-troubleshooting/troubleshoot-team-server-issues.md [hunk 2]
```diff
@@ -9,30 +9,35 @@ tags: ["version control", "troubleshoot", "team server"]
...
 
 ## 2 Troubleshooting Team Server App Network Settings
 
-Being unable to download the Team Server app can indicate that the security configuration of your company network is blocking access to `https://home.mendix.com` and `https://teamserver.sprintr.com/`.
+Being unable to download the Team Server app can indicate that the security configuration of your company network is blocking access to `https://home.mendix.com` and the Team Server itself.
+- `https://git.api.mendix.com/` for Git based apps.
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/version-control/version-control-troubleshooting/troubleshoot-team-server-issues.md]: What does this list indicate, are these URLs they need to whitelist? It is not clear from the paragraph what it lists.

File: content/en/docs/refguide/version-control/version-control-troubleshooting/troubleshoot-team-server-issues.md (hunk 1 above)
Comment (grootjans) [reply content/en/docs/refguide/version-control/version-control-troubleshooting/troubleshoot-team-server-issues.md:12]: I expect we will announce the actual dates/timelines in the next months. We will likely update the Version Control FAQ to reflect this.

File: content/en/docs/refguide/version-control/version-control-troubleshooting/troubleshoot-team-server-issues.md (hunk 2 above)
Comment (grootjans) [reply content/en/docs/refguide/version-control/version-control-troubleshooting/troubleshoot-team-server-issues.md]: If I add something like:
and the Team Server itself that is located at:

**Review by MariaShaposhnikova — APPROVED** [review]

**Pull request merged**